### PR TITLE
Feature: Update header to accept dynamic content [NP-761]

### PR DESCRIPTION
**Pull request by baseonmars**
This update allows for dynamic content to be inserted into the header and also adds a method for changing the styling of the header to support advisernet identity.

- [x] Update component for dynamic content
- [x] Update component for styling by variables
- [x] ~Update component to accept other logos~ documentation added instead
- [x] Update Visual Regression Tests
- [x] Document new variables
- [x] Update cucumber tests.

### CODE REVIEW

**Review by davidsauntson — CHANGES_REQUESTED** [review]
just a missing 💲 for the requested changes.

one thought is that we should remove `.cads-lang-cy` from the header logo and move to the public-website but not necessary if you want to get this over the line

File: scss/6-components/_header.scss [hunk 1]
```diff
@@ -29,19 +52,25 @@
     }
   }
 
-  .cads-header__account-links {
+  .cads-header__links {
```
[the remark this answers is not in the document]
Comment (luke-hill) [reply scss/6-components/_header.scss:55]: Rightio. No worries. We can test them just by running the tests against localhost:3000 though. So we can create the sister-PR alongside this if we're happy.

[the remark this answers is not in the document]
Comment (baseonmars) [reply scss/6-components/_header.scss:55]: I was going to mention this to Luke... tbh I am happy to get in the habit of making the changes myself? might need some hand holding to get started - are the instructions in the README up to date?

**Review by davidsauntson — APPROVED** [review]
👍

**Pull request merged**